code/modules/reagents/reagent_containers/cups/drinks.dm

```diff
@@ -366,27 +366,29 @@
 	icon_state = reagents.total_volume ? "water_cup" : "water_cup_e"
 	return ..()
 
-/obj/item/reagent_containers/cup/glass/sillycup/smallcarton
+/obj/item/reagent_containers/cup/glass/smallcarton
```

Comment:
Why not subtype of ``/obj/item/reagent_containers/cup/glass/bottle/juice``

Reply:
It looks like the only thing gained from doing that is letting a person kill themselves with a cardboard molotov. Which is funny, so sure. One moment.

Reply:
@JohnFulpWillard Done.